node/src/main.rs

```diff
@@ -84,7 +84,7 @@ async fn main() -> Result<()> {
     let filter = EnvFilter::builder()
         .with_default_directive(LevelFilter::INFO.into())
         .parse(format!(
-            "{tracing_level},h2::codec={network_tracing_level},{custom_directive}"
+            "{tracing_level},h2::codec={network_tracing_level},tower::buffer={network_tracing_level},tonic::transport={network_tracing_level},{custom_directive}"
```

Comment:
So, [this](https://github.com/MystenLabs/sui-operations/pull/54/files#diff-fabfa74fca59dbd3d68968f9e1e026e192fa95baa32297aa67b8c579720c2045R27) is the line used by the sui deployment.
I suggest: `h2={network_tracing_level},tower={network_tracing_level},hyper={network_tracing_level}`
I am not super sure about `tonic::transport`, because that's how we'll see disconnection errors.

Reply:
Nice, I can change that to be aligned between the codebases. Regarding the `tonic::transport` , disconnections are captured via `debug` statements from the framework already

Reply:
Sounds like the right idea. Stamping so you're unblocked to merge when you've edited this.